tau2_integration/domain_registry.py

```
@@ -0,0 +1,259 @@
+            
+            # Extract all tool functions
+            tools = {}
+            for name in dir(toolkit):
+                if name.startswith("_"):
+                    continue
+                attr = getattr(toolkit, name)
+                if callable(attr) and hasattr(attr, "__self__"):
+                    tools[name] = attr
+            
+            logger.info(f"Loaded {len(tools)} tools for {config.name} domain")
+            return tools
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)